t/balancer.t

```diff
@@ -595,3 +595,169 @@ ok
 qr/\[error\] .*? log_by_lua.*? failed to call: API disabled in the current context/
 --- no_error_log
 [alert]
+
+
+
+=== TEST 14: set_timeout (connect_timeout, send_timeout, read_timeout)
+--- http_config
+    lua_package_path "$TEST_NGINX_CWD/lib/?.lua;;";
+
+    upstream backend {
+        server 0.0.0.1;
+        balancer_by_lua_block {
+            print("hello from balancer by lua!")
+            local b = require "ngx.balancer"
+            b.set_timeout(1000, 1000, 1000)
+            assert(b.set_current_peer("127.0.0.1", tonumber(ngx.var.server_port)))
+        }
+    }
+--- config
+    location = /t {
+        proxy_pass http://backend;
+    }
+    location = /back {
+        echo "hello proxy timeout!";
+    }
+--- request
+    GET /t
+--- response_body:
```

Comment:
This is a syntax error. And you'll see the following error while running your `.t` file:

```
Extra lines not allowed in 'response_body' section at t/balancer.t
```

So you haven't run your new tests yourself?